pkgs/development/tools/diesel-cli/default.nix

```diff
@@ -9,11 +9,10 @@ assert lib.assertMsg (sqliteSupport == true || postgresqlSupport == true || mysq
 
 let
   inherit (lib) optional optionals optionalString;
-  features = ''
-    ${optionalString sqliteSupport "sqlite"} \
-    ${optionalString postgresqlSupport "postgres"} \
-    ${optionalString mysqlSupport "mysql"} \
-  '';
+  features =
+       optional sqliteSupport "sqlite"
```

Comment:
```suggestion
  features = optional sqliteSupport "sqlite"
```

Reply:
Fine with me, I added the line break, so that all the `optional`s are aligned.